src/handlebars/faq.handlebars

```diff
@@ -0,0 +1,258 @@
+{{> header title='FAQ | ' style1='<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.11.0/styles/obsidian.min.css">' }}
+
+<!-- TODO Style should match installation page, it doesn't quite... -->
+<main class="grey-bg">
+
+  <h1 class="margin-bottom large-title">AdoptOpenJDK Frequently Asked Questions</h1>
+
+  <div class="align-left support">
+
+    <div class="anchor">
+      <a href="#installers" class="anchor"><img src="dist/assets/anchor.png" alt="anchor icon"></a>
+      <h2 class="bold"><a id="installers">Installers</a></h2>
+    </div>
+    <div class="margin-bottom">
+      <p>
+        As well as basic tar and zip files that can be extracted onto your filesystem and installers from
+        the main download pages, AdoptOpenJDK also provides
+        <a href="https://adoptopenjdk.net/installation.html#installers">package repositories</a> for various
+        platforms. In addition to that various third-party package managers have support for installing
+        AdoptOpenJDK packages:
+
+        <ul> 
+        <li> <strong><a href="https://formulae.brew.sh/cask/adoptopenjdk">macOS brew</a></strong>
+             - Use <tt>brew cask install adoptopenjdk</tt> - maintained by the AdoptOpenJDK community at the
+             <a href="https://github.com/AdoptOpenJDK/homebrew-openjdk">openjdk-homebrew</a> repository.
+        <li> <strong><a href="https://sdkman.io/jdks">sdkman</a></strong> - <tt>sdk install java x.y.z-adpt</tt>
+        <li> <strong><a href="https://blog.io7m.com/2018/11/18/choosing-coffee.xhtml">coffeepick</a></strong> (via iojm)</strong>
+        <li> <strong><a href="https://ortussolutions.atlassian.net/browse/COMMANDBOX-911">commandBox</a></strong> - <tt>install java:openjdk8</tt>
+        <li> <strong><a href="https://chocolatey.org/packages/adoptopenjdk/">chocolatey</a></strong> - <tt>choco install adoptopenjdk</tt>
+        </li>
+      </p>
+    </div>
+
+    <div class="anchor">
+      <a href="#projects" class="anchor"><img src="dist/assets/anchor.png" alt="anchor icon"></a>
+      <h2 class="bold"><a id="projects">How do I get the source code for a release?</a></h2>
+    </div>
+
+    <p>The answer to this depends on which version you are interested in.
+    First, identify the major java release which will be the first number after
+    <code>jdk</code> in the release name. For example for
```

Comment:
For example, for <code>jdk8u232-b09</code> the release is <code>8</code>, for
    <code>jdk-11.0.5+10</code> it is <code>11</code></p>.

Reply:
Done